d3a_api_client/utils.py

```diff
@@ -11,11 +11,26 @@
 from sgqlc.endpoint.http import HTTPEndpoint
 
 from d3a_interface.constants_limits import JWT_TOKEN_EXPIRY_IN_SECS
+from d3a_interface.api_simulation_config.validators import validate_api_simulation_config
 from d3a_api_client.constants import DEFAULT_DOMAIN_NAME, DEFAULT_WEBSOCKET_DOMAIN, \
     CUSTOMER_WEBSOCKET_DOMAIN_NAME
 from d3a_interface.utils import get_area_name_uuid_mapping, key_in_dict_and_not_none, \
     RepeatingTimer
 
+from d3a_api_client.constants import DEFAULT_DOMAIN_NAME, DEFAULT_WEBSOCKET_DOMAIN, \
+    API_CLIENT_SIMULATION_ID
+
+DOMAIN_NAME_FROM_ENV = os.environ.get("API_CLIENT_DOMAIN_NAME", DEFAULT_DOMAIN_NAME)
+
+
```

Comment:
Why are you opting to have blank lines between each constant?

Reply:
just a habit, is there any PEP8 for that too ?

Reply:
Actually there is: https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Yes, so as far as I read there shouldn't be empty lines between constants.

Reply:
Yes, adapted accordingly!